### ISSUE TITLE
[.1] Shasum Has Not Been Updated Online for 171015 Release

### ISSUE THREAD

**Issue by landry314** [bitshares/bitshares-ui#585]
At least for the Mac OS version:

Online:
5ac099fe6f1ebd63e8d40b8c972a7b8636f8d1676f30dba5e0b84c01a4cef805

$ shasum -a 256 BitShares-2.0.171015.dmg
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855

**Comment by svk31** [comment]
Did you have the file downloaded completely before doing that? Cause it looks fine to me, tested on a different computer from the one I used originally:

```
$ sha256sum *171015*
5ac099fe6f1ebd63e8d40b8c972a7b8636f8d1676f30dba5e0b84c01a4cef805  BitShares-2.0.171015.dmg
```

**Comment by landry314** [comment]
Strange, I just downloaded it again from:
https://github.com/bitshares/bitshares-ui/releases

BitShares.Light.Wallet-2.0.171015.dmg
5ac099fe6f1ebd63e8d40b8c972a7b8636f8d1676f30dba5e0b84c01a4cef805

shasum -a 256 BitShares-2.0.171015.dmg
5ac099fe6f1ebd63e8d40b8c972a7b8636f8d1676f30dba5e0b84c01a4cef805

and they match this time. I am looking carefully at the old commands in the terminal window, it was different last time at 10pm EST for some reason. I also just tried from bitshares.org and it also matches. So, I guess there is no problem...
--

Except, on the bitshares.org website it still says:
BitShares 2 (July 7th, 2017 / GUI October 9th, 2017)

I also suggest you, @svk31, rename "BitShares.Light.Wallet-2.0.171015.dmg" to match the new names for the package... and all the packages.
